src/molgenis/capice_resources/train_data_creator/__main__.py

```diff
@@ -0,0 +1,182 @@
+        )
+        return {
+            **vkgl,
+            **clinvar,
+            **output
+        }
+
+    def run_module(self, arguments):
+        vkgl = self._read_pandas_tsv(
+            arguments['input_vkgl'],
+            [
+                Menums.CHROMOSOME.value,
+                Menums.START.value,
+                Menums.SUPPORT.value,
+                Menums.CLASSIFICATION.value
+            ]
+        )
+        parsed_vkgl = VKGLParser().parse(vkgl)
+        clinvar = self._read_vcf_file(
+            arguments['input_clinvar']
+        )
+        parsed_clinvar = ClinVarParser().parse(clinvar)
+        merge = merge_dataset_rows(parsed_clinvar, parsed_vkgl)
+        del clinvar, parsed_clinvar, vkgl, parsed_vkgl
+        gc.collect()
+
+        ConsensusChecker().check_consensus_clinvar_vkgl_match(merge)
+
+        DuplicateProcessor().process(merge)
+
+        SampleWeighter().apply_sample_weight(merge)
+
+        SVFilter().filter(merge)
+
+        train_test, validation = SplitDatasets().split(merge)
+
+        del merge
+        gc.collect()
+
+        return {
+            Genums.OUTPUT.value: arguments['output'],
+            Genums.TRAIN_TEST.value: train_test,
+            Genums.VALIDATION.value: validation
+        }
+
+    def export(self, output):
+        fake_vcf_header = [
+            '##fileformat=VCFv4.2',
+            '##contig=<ID=1,length=249250621,assembly=b37>',
+            '##contig=<ID=2,assembly=b37,length=243199373>',
+            '##contig=<ID=3,assembly=b37,length=198022430>',
+            '##contig=<ID=4,length=191154276,assembly=b37>',
+            '##contig=<ID=5,length=180915260,assembly=b37>',
+            '##contig=<ID=6,length=171115067,assembly=b37>',
+            '##contig=<ID=7,length=159138663,assembly=b37>',
+            '##contig=<ID=8,length=146364022,assembly=b37>',
+            '##contig=<ID=9,length=141213431,assembly=b37>',
+            '##contig=<ID=10,length=135534747,assembly=b37>',
+            '##contig=<ID=11,length=135006516,assembly=b37>',
+            '##contig=<ID=12,length=133851895,assembly=b37>',
+            '##contig=<ID=13,length=115169878,assembly=b37>',
+            '##contig=<ID=14,length=107349540,assembly=b37>',
+            '##contig=<ID=15,length=102531392,assembly=b37>',
+            '##contig=<ID=16,length=90354753,assembly=b37>',
+            '##contig=<ID=17,length=81195210,assembly=b37>',
+            '##contig=<ID=18,length=78077248,assembly=b37>',
+            '##contig=<ID=19,length=59128983,assembly=b37>',
+            '##contig=<ID=20,length=63025520,assembly=b37>',
+            '##contig=<ID=21,length=48129895,assembly=b37>',
+            '##contig=<ID=22,length=51304566,assembly=b37>',
+            '##contig=<ID=X,assembly=b37,length=155270560>',
+            '##contig=<ID=Y,length=59373566,assembly=b37>',
+            '##contig=<ID=MT,length=16569,assembly=b37>',
+            '##fileDate=20200320'
+        ]
```

Comment:
Might be better off as a [resource file](https://docs.python.org/3/library/importlib.resources.html) or class constant?